dlp/src/test/java/dlp/snippets/TriggersTests.java

```diff
@@ -104,12 +107,34 @@ public void setUp() {
     System.setOut(new PrintStream(bout));
   }
 
+  @AfterClass()
+  public static void cleanupTriggers() throws IOException {
+    try (DlpServiceClient dlpServiceClient = DlpServiceClient.create()) {
+      // List all job triggers
+      ListJobTriggersRequest listJobTriggersRequest =
+          ListJobTriggersRequest.newBuilder()
+              .setParent(ProjectName.of(PROJECT_ID).toString())
+              .build();
+      DlpServiceClient.ListJobTriggersPagedResponse response =
+          dlpServiceClient.listJobTriggers(listJobTriggersRequest);
+
+      // Loop through the response and delete each trigger
+      for (JobTrigger trigger : response.getPage().getValues()) {
+        DeleteJobTriggerRequest deleteJobTriggerRequest =
+            DeleteJobTriggerRequest.newBuilder().setName(trigger.getName()).build();
+        dlpServiceClient.deleteJobTrigger(deleteJobTriggerRequest);
+      }
+    }
+  }
```

Comment:
Ideally, you'll just clean up the ones you've created for testing.

Reply:
changed in latest commit